src/consensus/ledger_enclave.h

```diff
@@ -28,70 +28,58 @@ namespace consensus
      * Put a single entry to be written the ledger, when primary.
      *
      * @param entry Serialised entry
+     * @param globally_committable True is entry is signature transaction
      */
-    void put_entry(const std::vector<uint8_t>& entry)
+    void put_entry(const std::vector<uint8_t>& entry, bool globally_committable)
     {
       // write the message
-      RINGBUFFER_WRITE_MESSAGE(consensus::ledger_append, to_host, entry);
+      RINGBUFFER_WRITE_MESSAGE(
```

Comment:
Minor: This could be `put_entry(entry.data(), entry.size(), globally_committable)`.